keps/sig-storage/3762-persistent-volume-last-phase-transition-time/README.md

```diff
@@ -676,13 +651,20 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+Rollout should not fail and there should be no need for rollback as this enhancement only adds a new field.
+Also, rollback in terms of removal of this new field is not possible, once a PV is updated with the new field it can not
+be removed after disabling the feature.
```

Comment:
The field won't be deleted automatically, but we can manually reset it, right?

Reply:
Exactly, I'm adding a better explanation with examples.